SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -3022,7 +3022,6 @@
                 "python-packaging",
                 "python-pam",
                 "python-pbr",
-                "python-pip",
```

Comment:
don't we still provide python-pip for python3 ?

Reply:
Yep! It's part of the `python3` spec, so it won't appear in this list or in the `cgmanifest.json` file.

Reply:
The same sentiment applies to the `python3-setuptools` subpackage.